### PR TITLE
Remove OSXAdapter and replace with Java built-ins

### PR DESCRIPTION
**Pull request by gausie**
Everything is replaced with its equivalent. There was a line that allowed frames to be made fullscreenable but I think OSX has changed to allow that by default, and Java indeed doesn't offer a replacement as a result.

### CODE REVIEW

File: util/KoLmafia.importorder [hunk 1]
```diff
@@ -1,4 +1,3 @@
-0=apple.dts.samplecode.osxadapter
 1=ca.bcit.geekkit
```
Comment (heeheehee-kolmafia) [inline util/KoLmafia.importorder:1]: Should this be zero-indexed, or does it not matter?

**Review by heeheehee-kolmafia — APPROVED** [review]

**Comment by BadHorseMonkey** [comment]
I didn't see anything unusual when I checked out this branch and ran it.

Java broke taskbar badge support, but that was last year.

File: src/net/sourceforge/kolmafia/swingui/listener/LicenseDisplayListener.java [hunk 2]
```diff
@@ -9,10 +9,8 @@
   private static final String[] LICENSE_FILENAME = {
     "kolmafia-license.txt",
```
Comment (BadHorseMonkey) [inline src/net/sourceforge/kolmafia/swingui/listener/LicenseDisplayListener.java:10]: kolmafia-license.txt is missing, but I think that predates your change.  Should be in src/licenses, but it's no longer there and the page in About is blank.

**Review by BadHorseMonkey — APPROVED** [review]
Works.  Doesn't need to fix broken licenses.txt file, but it could.

File: util/KoLmafia.importorder (hunk 1 above)
Comment (gausie) [reply util/KoLmafia.importorder:1]: Nah it just traverses in order I think

**Comment by gausie** [comment]
> I didn't see anything unusual when I checked out this branch and ran it.
> 
> Java broke taskbar badge support, but that was last year.

All my badges work perfectly by the way!

**Pull request merged**